languages.toml

```diff
@@ -1676,7 +1676,11 @@ file-types = ["ron"]
 roots = []
 comment-token = "//"
 indent = { tab-width = 4, unit = "    " }
-grammar = "rust"
+grammar = "ron"
```

Comment:
since the name matches the language name we don't need to specify this key - it will default to this value